CONTRIBUTING.md

```diff
@@ -1,7 +1,14 @@
-Reach out to us on the ipywidgets gitter chat if you have any questions:
+# Contributing to bqplot
 
-Help / Documentation
---------------------
+## Governance and code of conduct
+
+Bqplot is subject to the [bqplot governance](https://github.com/bqplot/governance/blob/master/governance.md) and the [bqplot code of conduct](https://github.com/bqplot/governance/blob/master/code_of_conduct.md).
+
+## Questions
+
+Should you have any question, please do not hesitate to reach out to us on the [ipywidgets gitter chat](https://gitter.im/jupyter-widgets/Lobby).
```

Comment:
Should we maybe rename the bqplot Gitter channel and put a link to it here?

Reply:
At the moment, the bqplot gitter channel is in the bloomberg org...

Reply:
Yes, that is why I was mentioning renaming it, if that is doable :)

Reply:
Let's move this in another iteration.

Reply:
(This is mainly to have a reference to the CoC in the main repository.)